aspnetcore/tutorials/first-web-api.md

```diff
@@ -51,15 +51,23 @@ In Visual Studio, press CTRL+F5 to launch the app. Visual Studio launches a brow
 
 If you're using IE, you are prompted to open or save the *values.json* file.
 
-### Add support for Entity Framework Core
+## Add support for Entity Framework Core
+
+# [ASP.NET 1.x](#tab/aspnet1x)
 
 Install the [Entity Framework Core InMemory](https://docs.microsoft.com/ef/core/providers/in-memory/) database provider. This database provider allows Entity Framework Core to be used with an in-memory database.
 
 Edit the *TodoApi.csproj* file. In Solution Explorer, right-click the project. Select **Edit TodoApi.csproj**. In the `ItemGroup` element, add "Microsoft.EntityFrameworkCore.InMemory":
```

Comment:
~~Edit the *TodoApi.csproj* file. In Solution Explorer, right-click the project. Select **Edit TodoApi.csproj**.~~
Edit the *TodoApi.csproj* file by right-clicking the project in **Solution Explorer** and selecting **Edit TodoApi.csproj**.